DEVELOPER_GUIDE.md

```diff
@@ -55,7 +55,32 @@ Bound addresses will then be logged to the terminal :
 
 ## Run OpenSearch
 
-- Start a seperate terminal and navigate to the directory that OpenSearch has been cloned to using `cd OpenSearch`
+- Start a separate terminal and navigate to the directory that OpenSearch has been cloned to using `cd OpenSearch`.
+
+- Check if extensions directory exists in OpenSearch using `ls`.
```

Comment:
This is unclear. We need to specify about the build directory here and also can you mention the reason why do we have to create a new directory.
Something like: Every extension should have an extensions.yml file which should contain the metadata of the extension. To make the SDK look like as an extension with OpenSearch, need to have an extensions.yml file .....
(Phrase it better)